### DISCUSSION TITLE
Version of SPAdes

### DISCUSSION THREAD

**Discussion by chienfuwu** [ablab/spades#1004] [Q&A] [answered]
Hello!

I have a problem with the version of using the latest version 3.15.4. 
I have downloaded the latest package, but the version information still showed v 3.13.1.   With v 3.13.1, I can't work with rnaviral spades. Is there any solution? 
Thank you very much.

![1657885659592](https://user-images.githubusercontent.com/79143538/179340296-2ad5f1ca-38c0-4210-ba8e-e1b056bbabdc.jpeg)

**Comment by asl** [comment, marked as answer]
You are not running the version you downloaded. You are running the one installed globally. You need to run `spades.py` from the current directory, e.g. `./spades.py`.

Hope this helps.

**Reply by chienfuwu** [reply]
Thank you so much.
